>From 6d90001d274efe58d13bc8d40b34704bb98a55b0 Mon Sep 17 00:00:00 2001
From: Michael Stapelberg <michael@stapelberg.de>
Date: Tue, 7 May 2013 22:02:57 +0200
Subject: [PATCH] dh_installinit: remove duplicate systemd-tmpfiles call

This was left over in commit 469c74d0ac by mistake.

The code is in autoscripts/postinst-init-tmpfiles and gets pulled in
only when tmpfiles are actually present.
---
 autoscripts/postinst-init-nostart |    6 ------
 autoscripts/postinst-init-restart |    6 ------
 2 files changed, 12 deletions(-)

diff --git a/autoscripts/postinst-init-nostart b/autoscripts/postinst-init-nostart
index 6ccb483..7a1bd5e 100644
--- a/autoscripts/postinst-init-nostart
+++ b/autoscripts/postinst-init-nostart
@@ -1,9 +1,3 @@
 if [ -x "/etc/init.d/#SCRIPT#" ]; then
-	# In case this system is running systemd, we need to ensure that all
-	# necessary tmpfiles (if any) are created before starting.
-	if [ -d /run/systemd/system ] ; then
-		systemd-tmpfiles --create >/dev/null || true
-	fi
-
 	update-rc.d #SCRIPT# #INITPARMS# >/dev/null || #ERROR_HANDLER#
 fi
diff --git a/autoscripts/postinst-init-restart b/autoscripts/postinst-init-restart
index d27cc8d..35bba20 100644
--- a/autoscripts/postinst-init-restart
+++ b/autoscripts/postinst-init-restart
@@ -1,10 +1,4 @@
 if [ -x "/etc/init.d/#SCRIPT#" ]; then
-	# In case this system is running systemd, we need to ensure that all
-	# necessary tmpfiles (if any) are created before starting.
-	if [ -d /run/systemd/system ] ; then
-		systemd-tmpfiles --create >/dev/null || true
-	fi
-
 	update-rc.d #SCRIPT# #INITPARMS# >/dev/null
 	if [ -n "$2" ]; then
 		_dh_action=restart
-- 
1.7.10.4

